list_books.php

```diff
@@ -0,0 +1,34 @@
+<?php
+include 'database.php';
```

Comment:
Observations: The code directly uses raw SQL queries without any form of input validation or error handling.

Recommendations:
- Use prepared statements to prevent SQL injection.
- Consider refactoring the code to use an object-oriented approach with a `BookRepository` class.